### DISCUSSION TITLE
Backup

### DISCUSSION THREAD

**Discussion by Supersimmons** [cculianu/Fulcrum#243] [General]
Hello,
what are the best steps to make a Fulcrum backup so that it is consistent and certainly restoreable?
Thank you

**Comment by cculianu** [comment]
1. Stop Fulcrum completely and gracefully either with CTRL-C or sending it SIGINT (or SIGTERM).
2. Copy the data dir. This copy is your backup. 
3. To restore later, copy the dir back (or untar it or whatever).

**Comment by PrinceOfEgypt** [comment]
I run this as a weekly cron job to keep an up-to-date backup:

1. Stop fulcrum and anything else that depends on it
2. Incremental copy of the data directory
3. Restart fulcrum and the other services

`sudo systemctl stop fulcrum.service btcrpcexplorer.service mempool.service`
`sudo -u fulcrum rsync -arzq --delete /mnt/ssd/fulcrumdata/ /mnt/backups/fulcrumdata_weekly`
`sudo systemctl start fulcrum.service btcrpcexplorer.service mempool.service`

**Comment by berzekhunter** [comment]
On Windows 10, the only way to back up was by shutting down the Bitcoin node first and only then pressing Ctrl+C, waiting for the complete output. Only then did I copy the db folder to its backup location."
